tests/completion/snippets/task.yaml

```diff
@@ -20,4 +20,5 @@ execute: |
   #shellcheck disable=SC1090
   source "${SPREAD_PATH}/${SPREAD_SUITE}/${SPREAD_VARIANT}.vars"
   export _OUT0 _OUT1 _OUT2 _KEY1 _KEY2 _COMP
+  export XDG_DATA_DIRS="${XDG_DATA_DIRS:-}${XDG_DATA_DIRS:+:}/var/lib/snapd/desktop"
```

Comment:
`sudo ... -E -u test ...` is used here. sudo documentation claims that it would just fail if `-E` was not allowed. Unless I am not understanding correctly. Am I missing something?

Reply:
Ah, I missed -E. It's fine then.